mock/el8.cfg

```diff
@@ -2,7 +2,7 @@ config_opts['root'] = 'epel-8-x86_64'
 config_opts['target_arch'] = 'x86_64'
 config_opts['legal_host_arches'] = ('x86_64',)
 
-config_opts['chroot_setup_cmd'] = 'install tar gcc-c++ redhat-rpm-config redhat-release which xz sed make bzip2 gzip gcc coreutils unzip shadow-utils diffutils cpio bash gawk rpm-build info patch util-linux findutils grep'
+config_opts['chroot_setup_cmd'] = 'install tar gcc-c++ redhat-rpm-config redhat-release which xz sed make bzip2 gzip gcc coreutils unzip shadow-utils diffutils cpio bash gawk rpm-build info patch util-linux findutils grep --exclude=devtoolset*'
```

Comment:
I'm not sure devtoolset ships on EL8. @ehelms @pcreech @zjhuntin @evgeni any idea?

Reply:
Looks like it is not available on el8 at the moment.

Reply:
Not to my knowledge there isn't.